src/client/Microsoft.Identity.Client/MsalError.cs

```diff
@@ -1149,5 +1149,29 @@ public static class MsalError
         /// A cryptographic exception occurred when trying to use the provided certificate
         /// </summary>
         public const string CryptographicError = "cryptographic_error";
+
+        /// <summary>
+        /// <para>What happened?</para>mTLS Proof of Possession (mTLS PoP) is configured but a region was not specified.
+        /// <para>Mitigation</para>Ensure that the AzureRegion configuration is set when using mTLS PoP as it requires a regional endpoint.
+        /// </summary>
+        public const string MtlsPopWithoutRegion = "mtls_pop_without_region";
+
+        /// <summary>
+        /// <para>What happened?</para> mTLS Proof of Possession (mTLS PoP) is configured but a certificate was not provided.
+        /// <para>Mitigation</para> Ensure that a valid certificate is provided in the configuration when using mTLS PoP as it is required for secure authentication.
+        /// </summary>
+        public const string MtlsCertificateNotProvided = "mtls_certificate_not_provided";
+
+        /// <summary>
+        /// <para>What happened?</para> mTLS Proof of Possession (mTLS PoP) requires a specific Azure region to be specified.
```

Comment:
it can also happen if auto-discovery is configured but it fails.

Reply:
it is handled, by `public const string RegionRequiredForMtlsPopMessage = "Regional auto-detect failed. mTLS Proof-of-Possession requires a region to be specified, as there is no global endpoint for mTLS. See https://aka.ms/msal-net-pop for details.";`